uint32 sol_status
uint32 pos_type

float64 pos_x
float64 pos_y
float64 pos_z

float32 std_pos_x
float32 std_pos_y
float32 std_pos_z

uint32 v_sol_status
uint32 vel_type

float64 vel_x
float64 vel_y
float64 vel_z

float32 std_vel_x
float32 std_vel_y
float32 std_vel_z
